src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -190,10 +200,14 @@ const SelectPanelHeader: React.FC<React.PropsWithChildren> = ({children, ...prop
           {/* heading element is intentionally hardcoded to h1, it is not customisable 
             see https://github.com/github/primer/issues/2578 for context
           */}
-          <Heading as="h1" sx={{fontSize: 14, fontWeight: 600}} {...props}>
+          <Heading as="h1" id={`${panelId}--title`} sx={{fontSize: 14, fontWeight: 600}} {...props}>
             {title}
           </Heading>
-          {description ? <Text sx={{fontSize: 0, color: 'fg.muted', display: 'block'}}>{description}</Text> : null}
+          {description ? (
```

Comment:
I think it would be nice to have a story with this property? I couldn't find it. Did I miss it?